libsplinter/src/migrations/diesel/sqlite/migrations/2020-12-08-121200-oauth-create-inflight-requests/up.sql

```diff
@@ -0,0 +1,21 @@
+---- Copyright 2018-2020 Cargill Incorporated
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+-- -----------------------------------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS oauth_inflight_request (
+    id                  TEXT            PRIMARY KEY,
+    pkce_verifier       TEXT            NOT NULL,
+    client_redirect_url TEXT            NOT NULL,
+    last_updated        TIMESTAMP       DEFAULT CURRENT_TIMESTAMP NOT NULL
```

Comment:
Would `created` be more accurate, since these entries never really get updated?

Reply:
renamed